tests/test_notanorm.py

```diff
@@ -509,6 +534,7 @@ def slow_q():
 
 
 def test_readonly_fail(db, db_name: str):
```

Comment:
Might be worth adding RO support -- nice to enforce we're not writing to DBs we don't intend to.

Reply:
ok

Reply:
done